packages/ti_threatconnect/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.2.3"
```

```suggestion
- version: "1.3.0"
```
(enhancements bump minor)

---

This has not been done.